pkg/didexchange/invitation/invitation.go

```diff
@@ -0,0 +1,85 @@
+// +build js,wasm
+
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package invitation
+
+import (
+	"encoding/json"
+	"fmt"
+	"syscall/js"
+	"time"
+
+	"github.com/hyperledger/aries-framework-go/pkg/client/didexchange"
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/common/service"
+)
+
+// RegisterHandleInvitationJSCallback register handle invitation js callback
+func RegisterHandleInvitationJSCallback(didexchangeClient *didexchange.Client) {
+	statusCh := make(chan service.StateMsg, 10)
```

Comment:
Any reason for 10? Would it make sense to parameterize?

Reply:
I will add TODO